inprocess/src/test/java/io/grpc/inprocess/InProcessTransportTest.java

```diff
@@ -83,7 +83,7 @@ protected String testAuthority(InternalServer server) {
   protected ManagedClientTransport newClientTransport(InternalServer server) {
     return new InProcessTransport(
         new InProcessSocketAddress(TRANSPORT_NAME), GrpcUtil.DEFAULT_MAX_HEADER_LIST_SIZE,
-        testAuthority(server), USER_AGENT, eagAttrs(), false);
+        testAuthority(server), USER_AGENT, eagAttrs(), false, -1);
```

Comment:
You can leave this as it was since the default will be -1.  You should also add a test where you use create the InProcessTransport with a positive number (like 100) and verify that the reported length is what you expected.